qabel_id/settings/base_settings.py

```diff
@@ -76,8 +76,8 @@
 AXES_COOLOFF_TIME = datetime.timedelta(minutes=1)
 AXES_LOGIN_FAILURE_LIMIT = 5
 
-ACCOUNT_EMAIL_CONFIRMATION_ANONYMOUS_REDIRECT_URL = 'https://qabel.de'
-ACCOUNT_EMAIL_CONFIRMATION_AUTHENTICATED_REDIRECT_URL = 'https://qabel.de'
+ACCOUNT_EMAIL_CONFIRMATION_ANONYMOUS_REDIRECT_URL = 'https://accounting.qabel.org'
+ACCOUNT_EMAIL_CONFIRMATION_AUTHENTICATED_REDIRECT_URL = 'https://accounting.qabel.org'
```

Comment:
This would lead to a 404 since the accounting server doesn't have any view at /. The redirect to qabel.de was intentional.